README.md

```
@@ -5,132 +5,282 @@
 ```bash
-git clone git@github.com:ondrovic/nexus-mods-scraper.git
-cd nexus-mods-scraper
-go run nexus-mods-scraper.go
+./nexus-mods-scraper extract [flags]
 ```
 
-## Usage
+#### Example Output
 
-### Scrape Command
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)